infrastructure/resource_group_level/main.bicep

```diff
@@ -950,7 +947,7 @@ module frontDoorTfm 'modules/front-door-tfm.bicep' = {
 
 var tfmUiUrl = 'https://${frontDoorTfm.outputs.defaultHostName}'
 
-module tfmApiConnectionStrings 'modules/webapps/trade-finance-manager-api-connection-strings.bicep' = {
+module tfmApiConnectionStrings 'modules/webapps/trade-finance-manager-api-calculated-variables.bicep' = {
   name: 'tfmApiConnectionStrings'
```

Comment:
The name of the module and the variable should be updated too - they are both currently `'tfmApiConnectionStrings'`